src/lime/graphics/utils/ImageDataUtil.hx

```diff
@@ -421,8 +422,50 @@ class ImageDataUtil {
 		image.version++;
 	}
 
-	public static function gaussianBlur(image:Image, sourceImage:Image, sourceRect:Rectangle, destPoint:Point, blurX:Float = 4, blurY:Float = 4,
-			quality:Int = 1, strength:Float = 1) {
+	static var __tempColorMatrix:Float32Array;
+
+	public static function coloredGaussianBlur(image:Image, sourceImage:Image, sourceRect:Rectangle, destPoint:Point, blurX:Float, blurY:Float,
```

Comment:
This function is needed for blur/dropshadow filters, as gaussian blur can only work correctly with non-premultiplied-alpha pixels and if we do a "proper" premultiplied colortransform, the information about color will be lost in transparent areas which will mess up colors for blur as transparent pixels will be considered black.